### DISCUSSION TITLE
Collection table data not showing updated when editing mongo directly

### DISCUSSION THREAD

**Discussion by ericuldall** [payloadcms/payload#4503] [Announcement]
### Describe the Bug

I have a query in an order collection that changes my stock for inventory items using $inc
```
              const inventory = payload.db.collections.inventory
              await inventory.findByIdAndUpdate(
                inventoryItem.id,
                {
                  $inc: { stock: quantityDiff }
                }, {
                  new: true
                }
              ).exec();
```

The fields are updated correctly when looking at the edit form for that collection item, but the data is showing the previous value when looking at the list of collection items.

Is there some extra step I need to take to fully commit that change?

### To Reproduce

Enable versions, then edit mongo data directly.

### Payload Version

2

**Comment by DanRibbens** [comment]
Okay, since you mentioned you have versions enabled, I know what is going on.

The list view is displayed by querying a secondary collection that is named from your collection slug, prefixed with `_versions_`, so `_inventory_versions` in your case. Only published documents end up in the natural db collection and this is not the data that is used in the list view. 

You might want to update inventory using the Paylaod local API, you will have to read and update the document to increment the stock in code. The upside is that your versions will be updated in the database as one might expect.

If you don't care about maintaining the version history of the stock value specifically, you could maintain both collections versions and collection how you have done it. You can access the latest version for a document by querying on the `latest: {$eq: true}` and you can get the version collection from: `payload.db.versions.inventory`.

I am going to convert this issue to a discussion since there isn't anything to fix at this time.

**Reply by ericuldall** [reply]
The downside to the read/update pattern is that it will introduce race conditions. It's not really something i want to risk.

I'll probably go with your second suggestion and update the latest _versions document as well.
The version history is mostly for auditing actions made by team members directly editing records so we don't need a version for every instance of this stock change hook.

**Comment by TheTimeWalker** [comment]
Hey there, just to note, I got this as an announcement on GitHub. Is this intended?

**Comment by GeorgeHulpoi** [comment]
> Hey there, just to note, I got this as an announcement on GitHub. Is this intended?

@TheTimeWalker I think it was a mistake.
